test/dsl/dsl_basics.jl

```diff
@@ -2,11 +2,15 @@
 
 ### Fetch Packages and Set Global Variables ###
 
+# Fetch packages.
 using Catalyst, ModelingToolkit
+
+# Set creates the `t` independent variable.
```

Comment:
We already merged the `t = default_t()` change. We can always revert later, this is relatively easy to do.